addons-l10n/en/delete-others.json

```
@@ -0,0 +1,8 @@
+{
+  "delete-others/deleteOthers": "delete others",
+  "delete-others/confirmTitle": "Delete Others",
+  "delete-others/infoCostume": "Are you sure you want to delete all other costumes?\nThe deleted costumes can be restored by going to Edit > Restore Costumes",
+  "delete-others/infoSound": "Are you sure you want to delete all other sounds?\nThe deleted sounds can be restored by going to Edit > Restore Sounds",
```

```suggestion
  "delete-others/infoCostume": "Are you sure you want to delete all other costumes?\nThe deleted costumes can be restored by going to Edit \u2192 Restore Costumes.",
  "delete-others/infoSound": "Are you sure you want to delete all other sounds?\nThe deleted sounds can be restored by going to Edit \u2192 Restore Sounds.",
```
`\u2192` is a right arrow (→).